openmeetings-db/src/main/java/org/apache/openmeetings/db/dao/basic/ConfigurationDao.java

```diff
@@ -573,6 +574,11 @@ private void addCspRule(CSPHeaderConfiguration cspConfig, CSPDirective key, Stri
 	public void updateCsp() {
 		setGaCode(getString(CONFIG_GOOGLE_ANALYTICS_CODE, null));
 
+		if (!getBool(CONFIG_CSP_ENABLED, true)) {
+			WebApplication.get().getCspSettings().blocking().disabled();
```

Comment:
@CkNoSFeRaTU Can I ask you to add **Very Visible** warning here?

Reply:
Sure, added. Is it big enough?